README.rst

```diff
@@ -70,9 +70,74 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Clone the *coala-html* repository
 
-Note: You need to have bower installed before you may install the python packages.
+::
+
+  $ git clone https://github.com/coala/coala-html.git
+
+2. Install latest version of **Node.js**
+
+- *Configure Node.js PPA*
+
+::
+
+    $ sudo apt-get install curl python-software-properties
+    $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+- *Install Node.js and npm*
+
+::
+
+  $ sudo apt-get install nodejs
+
+- *Check Node.js and npm version*
+
+  You can find more details about current version on node.js `official website
+  <https://nodejs.org/>`__.
+
+::
+
+    $ node -v
+    v11.6.0
+
+    $ npm -v
+    6.5.0
```

Comment:
this would have to be updated frequently, can't we skip the output?
Or maybe we should give the user a range of versions that are acceptable (lower limit would be good too).